pkgs/development/tools/halfempty/default.nix

```diff
@@ -7,8 +7,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "googleprojectzero";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-YGq6fneAMo2jCpLPrjzRJ0eeOsStKaK5L+lwQfqcfpY=";
+    rev = "ad15964d0fcaba12e5aca65c8935ebe3f37d7ea3"; # Unreleased fix for https://github.com/googleprojectzero/halfempty/issues/17
```

Comment:
This should also change the version to `unstable-YYYY-MM-DD`

If the fix is just a single patch, I would prefer to just fetchpatch the patch.

Reply:
Done